modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -1904,10 +1904,18 @@ private void _notifySubscribers(
 		boolean sendEmailEntryUpdated = GetterUtil.getBoolean(
 			serviceContext.getAttribute("sendEmailEntryUpdated"));
 
+		Map<String, Serializable> attributes = serviceContext.getAttributes();
```

Comment:
Hi @laszlopap 

With `serviceContext.getAttribute("sendEmailEntryUpdated")` you can check if the attribute is null, you do not need to get all the attributes and you can reuse it with the `GetterUtil.getBoolean()` on line 1904